external-crates/move/crates/move-analyzer/src/symbols.rs

```diff
@@ -394,6 +394,13 @@ pub struct CursorContext {
     pub loc: Loc,
 }
 
+#[derive(Clone, Debug, Copy)]
+pub enum ChainCompletionTarget {
```

Comment:
Happy to change if from `Target`, though it does not sound that bad to me as we are indeed aiming for a function or a type with the completed chain. Also `Context` sounds a bit confusing to me in turn, as it suggests something that surrounds the chain rather than what the chain represents. Not sure what the best compromise here could be (`ChainEntityType/Kind`?)

Reply:
I like `kind` a lot, and it's consistent with our verbage in the compiler.